src/components/Link/tests/Link.test.tsx

```diff
@@ -81,4 +82,42 @@ describe('<Link />', () => {
       expect(link.find(Icon).exists()).toBe(false);
     });
   });
+
+  describe('monochrome link', () => {
+    it('outputs a monochrome unstyled link if rendered within a banner', () => {
+      const link = mountWithAppProvider(
+        <BannerContext.Provider value>
```

Comment:
Could we replace this was `Banner`?

```tsx
const link = mountWithAppProvider(
    <Banner>
        <Link url="https://examp.le">Some content</Link>
     </Banner>,
)
```

We're currently testing the context opposed to the result of rendering within a Banner